File: in_Java/0771 Jewels and Stones.java
class Solution {
    public int numJewelsInStones(String J, String S) {
        int res = 0;
        List<Character> l = new ArrayList<>();
        for(char c : J.toCharArray()) l.add(c);
        for(char c : S.toCharArray()){
            if(l.contains(c)) res++;
        }
        return res;
    }
}

File: in_Python/0771 Jewels and Stones.py
class Solution(object):
    def numJewelsInStones(self, J, S):
        """
        :type J: str
        :type S: str
        :rtype: int
        """
        res = 0
        for x in S:
            if x in J:
                res += 1
        return res